storage/libradb/src/lib.rs

```diff
@@ -9,6 +9,8 @@
 //! It relays read/write operations on the physical storage via [`schemadb`] to the underlying
 //! Key-Value storage system, and implements libra data structures on top of it.
 
+#[macro_use]
+extern crate prometheus;
```

Comment:
this is pre-rust 2018 convention, check this out:
https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html

Reply:
@davidiw Prometheus doesn't have a release which is 2018 compatible yet :( 
I will update this when we have a new Prometheus release

Reply:
I've been anxiously waiting for another release from them but looks like we still have to wait longer :/